front/app/modules/commercial/representativeness/admin/components/ChartCard/utils.ts

```diff
@@ -0,0 +1,40 @@
+import { RepresentativenessRow } from '.';
+import { Moment } from 'moment';
+import { format } from 'd3-format';
+
+interface TooltipProps {
+  dataKey?: 'actualPercentage' | 'referencePercentage';
+  payload?: RepresentativenessRow;
+}
+
+export const formatPercentage = (percentage: number) => `${percentage}%`;
+
+export const formatThousands = format(',');
+
+export const formatTooltipValues = (_, __, tooltipProps?: TooltipProps) => {
```

Comment:
This is used by `recharts`, which is why it looks so weird. See [docs](https://recharts.org/en-US/api/Tooltip#formatter)